core/src/main/scala/kafka/server/QuotaFactory.scala

```diff
@@ -37,7 +37,8 @@ object QuotaFactory extends Logging {
 
   object UnboundedQuota extends ReplicaQuota {
     override def isThrottled(topicPartition: TopicPartition): Boolean = false
-    override def isQuotaExceeded(): Boolean = false
+    override def isQuotaExceeded: Boolean = false
```

Comment:
Looks like the build failed because the usages of `isQuotaExceeded()` weren't updated?

Reply:
Oops, not sure how that happened. Will fix.